graphql_compiler/scripts/generate_test_sql/species.py

```diff
@@ -0,0 +1,28 @@
+import random
+
+from .utils import create_vertex_statement, get_uuid
+
+
+random.seed(0)
```

Comment:
Importing a new file without doing anything with it shouldn't reset your random seed. I suspect this isn't even doing what you wanted -- all these are imported at the beginning of the generator program, which means there are back-to-back `random.seed(0)` calls with no random generations in between. I suspect that's not what you had in mind.